wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1855,3 +1857,38 @@ class StringConstantRedefinedViolation(ASTViolation):
 
     error_template = 'Found alphabet as strings: {0}'
     code = 447
+
+
+@final
+class MutableClassAttributesViolation(ASTViolation):
+    """
+    Forbids mutable class attributes.
+
+    Reasoning:
+        Class attributes should be immutable.
+
+    Solution:
+        Use immutable types for class attributes.
+
+    Example::
+
+        # Correct:
+        class Test(object):
+            field = ('field1',)
+
+        class Test(object):
```

Comment:
Just one correct is fine. And the same for the wrong example.

Reply:
done